### ISSUE TITLE
Couple of Issues - Bugs, demographics

### ISSUE THREAD

**Issue by torn1147** [settintotrieste/Victoria-2-Cold-War-Enhancement-Mod-CWE#843]
Hey, so I did another playthrough and I wanted to share a couple of issues I noticed:

1. So a nuclear war took place and I was the victor in it. The other country(India) got nuked very heavily and almost all it's provinces had the nuclear strike modifier. However, after the war ended and I got the mandates over India's dismantled substates, the "nuclear strike" modifier disappeared from the provinces belonging to the mandates, although the fallout graphical effect remained there. Like, 50ish years later, whenever the "low impact from nuclear strike: modifier is supposed to disappear - it did so for all the provinces that had remained part of India. However, the provinces that were part of the mandates missing the modifier already remained stuck with the fallout graphical effect perpetually. Not too big an issue, but kinda led to a weird demographics situation for reunited India overall well after the nuclear war.

2. While playing as punjab(released from India), I had an issue where my pops seemingly weren't promoting properly and focuses were hardly working. I took a closer look at the demographics and the Soldier/Executive/Officer/Worker pops were all exclusively Hindu, while the primary culture was Punjabi and all farmers/bureaucrats. So the punjabi pops had nothing to promote to leading to a weird situation where I could only rely on a non-accepted pop(already being assimilated to pops belonging to punjabi culture and thus in decline) for soldiers and workers. Don't recall what the other pop types(intellectuals etc) looked like, but some might've been missing as well.

3. East Punjab's(the part belonging to India) demographics ingame reflect a 60-39-1 split between the Muslim-Hindu-Sikh religions while IRL this seems to be closer to 2-39-59 ish. [Culture-wise](https://en.wikipedia.org/wiki/Demographics_of_Punjab,_India), 90% of punjabi households recorded punjabi as their mother language while only 8% recorded Hindu as a mother tongue, while in-game starts off closer to 60-40. I know the numbers I mentioned are from 2010's, but punjab's demographics have barely shifted since Partition. Not a big issue. Just hoped to bring it to notice.

4. Regarding the nuclear strike modifier's on the provinces themselves, I think they should be nerfed atleast a bit. Any province that gets even the "low damage" modifier is reduced to a couple hundred people in total 50 years later from previously 100k's or millions, which seems a bit extreme. Just a suggestion.

Also, thank you for this amazing mod and your terrific work!!

**Comment by settintotrieste** [comment]
Thanks for your feedback!

1. There is a hardcoded limitation in the victoria game engine that prevents province modifiers from being carried over after a province has changed owners so I'm afraid nothing can be done to fix this.
2. I'm afraid this also looks like a hardcoded limitation that I cannot fix.
3. Okay I'll try and fix this if I can.
4. I'll think about it as low damage in this mod is supposed to represent one or two thermonuclear strikes on the province which would likely mess up the province for a long time.

**Comment by torn1147** [comment]
I made a fork that fixes the demographic situation in Indian Panjab based on the 1951 census. Also increased the soldier & officer %age of punjabi pops in all scenarios to reflect the historical predominance of sikhs in both the Indian and Raj's military. Adding some more punjabi pop types also fixed issue no.2 that I encountered.

https://github.com/torn1147/Victoria-2-Cold-War-Enhancement-Mod-CWE

Also, I want to remove panjab's cores from pakistani punjab, as punjabi muslims have historically never been interested in an independent panjab. It was mostly a punjabi sikh cause. However, I'll add a decision for panjab to re-add these cores on pakistani panjab at an infamy cost if the right party is in charge. Let me know how you feel about this.
